src/track/trackmetadatataglib.cpp

```diff
@@ -225,8 +225,15 @@ bool parseBpm(TrackMetadata* pTrackMetadata, QString sBpm) {
     DEBUG_ASSERT(pTrackMetadata);
 
     bool isBpmValid = false;
-    const double bpmValue = Bpm::valueFromString(sBpm, &isBpmValid);
+    double bpmValue = Bpm::valueFromString(sBpm, &isBpmValid);
     if (isBpmValid) {
+        //Some software uses (or used) to write decimated values without comma,
+        //so the number reads as 1352 or 14525 when it is 135.2 or 145.25
+        if (bpmValue > 5000.0) {
```

Comment:
Ok, that's fine. I will add the log message.

The problem with this tag is that when Mixxx reads it, the bpm analyzer does not run for such tracks (it thinks it is already calculated and one has to tell Mixxx to reset the value to force the analyzer to run).
As such, i tried, to make the problem less problematic.

And since you mention writing the tags back, i didn't modify that part (so it still writes it in the standard way), but we may need to rethink it. 

If a file has 1354  and we read 135.4 and then we store 135, and somehow the user rescans or reloads the data from the file, then it will become 135 and even if this looks like a small change, the beatgrid goes all out of place and given and we are back to the scenario of requiring manual deletion and reanalyzing.